src/styles/App.scss

```diff
@@ -1,3 +1,48 @@
+/* Custom Scrollbar */
+::-webkit-scrollbar
+{
+  width: 12px;
+}
+
+::-webkit-scrollbar-thumb
+{
+  background-color: #f37121;	
+}
+::-webkit-scrollbar-thumb:hover
+{
+  background-color: #ee9b00;	
+}
+
+::-webkit-scrollbar-button:single-button {
+    background-color: rgb(255, 253, 253);
+    display: block;
+    background-size: 10px;
+    background-repeat: no-repeat;
+}
+
+/* Up Arrow*/
+::-webkit-scrollbar-button:single-button:vertical:decrement {
+    height: 16px;
+    width: 16px;
+    background-position: center 4px;
+    background-image: url("../assets/uparrow.png");
+    }
+
+/* Down Arrow*/
+::-webkit-scrollbar-button:single-button:vertical:increment {
+    height: 16px;
+    width: 16px;
+    background-position: center 2px;
+    background-image: url("../assets/downarrow.png");
+}
+
+/*Hiding Scrollbar on Mobile Screens*/
+
+@media screen and (max-width: 768px) {
+  ::-webkit-scrollbar {
+    display: none;
+  }
+}
```

Comment:
Good work. Can we please have this indented properly according to prettier rules?

Reply:
Done 🙂